src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -196,6 +196,10 @@
 
   static final String UNDEFINED_LABEL = "undefined label \"%s\"";
 
+  static final String EXPOSE_DEPRECATED_WARNING =
```

Comment:
It should be suppressible. What's the example program that you tried? Can you try suppressing other warnings in IRFactory and see what happens?